README.md

```diff
@@ -24,15 +24,15 @@ The latest version of Kitura-CredentialsHTTP requires **Swift 3.0**. You can dow
 ## API
 
 ### Basic authentication
-To create an instance of CredentialsHTTPBasic plugin, a `UserProfileLoader` function and an optional realm should be passed to the constructor:
+To create an instance of `CredentialsHTTPBasic` plugin, use the `verifyPassword` initializer that takes an optional realm:
 ```swift
-public init (userProfileLoader: UserProfileLoader, realm: String?=nil)
+public init (verifyPassword: @escaping VerifyPassword, realm: String?=nil)
 ```
-`userProfileLoader` function should be of type:
+the `verifyPassword` argument is a typealias:
 ```swift
-public typealias UserProfileLoader = (userId: String, callback: (userProfile: UserProfile?, password: String?)->Void) -> Void
+public typealias VerifyPassword = (userId: String, password: String, callback: @escaping (UserProfile?) -> Void) -> Void
 ```
-It receives user id, and calls `callback` with `UserProfile` instance and password that correspond to the user id, or `nil` if such user id doesn't exist.
+It receives a userId and password, and it should invoke the `callback` with a `UserProfile` if the user exists or `nil` if the user doesn't exist.
 
```

Comment:
If the user exists and the password matches user's password.

I like the comment in the code, maybe just put it here?
```swift
/// Type alias for the callback that verifies the userId and password and if
/// the authentication pair verifies, then a user profile is returned.
```